LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) Foxglove Technologies Inc
+Copyright (c) 2021 Foxglove Technologies Inc
```

Comment:
cc @amacneil

Reply:
See the Copyright Compendium, Chapter 2200, section 2205.1(A) (https://www.copyright.gov/comp3/chap2200/ch2200-notice.pdf).

>  Year of Publication
> As a general rule, the notice on visually perceptible copies and on phonorecords of
> sound recordings must include the year of first publication. The notice on copies of a
> compilation or derivative work incorporating previously published material only
> requires the year of first publication for the compilation or derivative work. 17 U.S.C. §§
> 401(b)(2), 402(b)(2).
> A notice may be accepted if the year of publication is presented in any of the following forms:
> • Arabic numerals (e.g., 1981).
> • Abbreviations of Arabic numerals (e.g., ’81).
> • Roman numerals (e.g., MCMLXXXI).
> • Numeric words (e.g., Nineteen Hundred Eighty-One).
> • The year of first publication followed by multiple year dates (e.g., 1981, 1982, 1983).

Reply:
I'm not a lawyer and don't feel strongly about this, but as a counter point, FB doesn't put the year on their open source copyright notices: https://github.com/facebook/react/pull/13593

Keeping these up to date is a small but annoying chore, so it's kinda nice to not do it if it's not strictly necessary?